api/src/org/apache/cloudstack/api/command/user/iso/GetUploadParamsForIsoCmd.java

```diff
@@ -32,12 +32,14 @@
 import org.apache.cloudstack.api.response.GuestOSResponse;
 import org.apache.cloudstack.api.response.ZoneResponse;
 import org.apache.cloudstack.context.CallContext;
+import org.apache.cloudstack.acl.RoleType;
 
 import java.net.MalformedURLException;
 
 @APICommand(name = GetUploadParamsForIsoCmd.APINAME,
         description = "upload an existing ISO into the CloudStack cloud.",
         responseObject = GetUploadParamsResponse.class, since = "4.13",
```

Comment:
it says 4.13 here, should it say 4.11?

Reply:
@Slair1 Yes, it should be 4.11 according to our version, I think it is ported from future version from main so it is 4.13. Shall I change it to 4.11?

Reply:
@Slair1 Confirm you'd like 4.13 changed to 4.11 here?

Reply:
Done changes